tests/models/test_python_api.py

```diff
@@ -62,6 +62,7 @@ def predict(self, context, model_input):
         mlflow.pyfunc.log_model(
             artifact_path="model",
             python_model=TestModel(),
+            extra_pip_requirements=["pytest"],
```

Comment:
it looks like "_pytest" is loaded into the environment, so as a quick fix i added this line to make the tests pass.